dist/common/scripts/scylla_util.py

```diff
@@ -378,13 +378,7 @@ def is_azure_instance():
             addrlist = socket.getaddrinfo('metadata.azure.internal', 80)
         except socket.gaierror:
             return False
-        for res in addrlist:
-            af, socktype, proto, canonname, sa = res
-            if af == socket.AF_INET:
-                addr, port = sa
-                if addr == "169.254.169.254":
-                    return True
-        return False
+        return True
```

Comment:
Oh, it's removed here. Don't introduce something in patch 1 and remove it in patch 3, unless your goal is to annoy the reviewer.

Reply:
will squash, sorry, was trying to make the change after review obvious